js/client/modules/@arangodb/testsuites/recovery_cluster.js

```diff
@@ -119,6 +121,7 @@ function runArangodRecovery (params) {
   // arangosh has different name for parameter :(
   additionalTestParams['javascript.execute'] =  params.script;
   additionalTestParams['javascript.run-main'] = true;
+  additionalTestParams['server.request-timeout'] = '60';
```

Comment:
I guess we should do this only if params.setup is true? 
And wouldn't this introduce a problem if setup query will need to run morethan a minute? But the latter we could just document as a limitation for setup.

Reply:
yes, absolutely!